ckan/lib/helpers.py

```diff
@@ -2344,6 +2344,35 @@ def mail_to(email_address, name):
     return html
 
 
+@core_helper
+def get_base_public_folder(app_conf):
```

Comment:
Well that was how it was done before I added the helper functions. But in that way how are we going to raise an exception for invalid value? We can't just raise exceptions everywhere where it needs to be read from config.